### PR TITLE
Updated description of use_directory_urls

### PR DESCRIPTION
**Pull request by ronzulu**
Fixes issue #3797

### CODE REVIEW

File: docs/user-guide/configuration.md [hunk 1]
```diff
@@ -668,34 +668,33 @@ setting `use_directory_urls` to `true` or `false`.
 
 `use_directory_urls: false`
 
-This setting is needed when the documentation is hosted on systems that can't 
-access the file `X\index.html` when given the URL `X`. When set to `false`, 
-no additional `X` directory is created, and the file is simply stored as `X.html`. 
+This setting is needed when the documentation is hosted on systems that can't
+access the file `X\index.html` when given the URL `X`. When set to `false`,
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:672]: I suggest using a less platform-specific separator such as `/` which generally works for both Windows and Linux in Python.

```suggestion
access the file `X/index.html` when given the URL `X`. When set to `false`,
```

File: docs/user-guide/configuration.md [hunk 2]
```diff
@@ -668,34 +668,33 @@ setting `use_directory_urls` to `true` or `false`.
...
-| Source file      | Generated File     | URL Format          |
-| ---------------- | ------------------ | ------------------- |
-| index.md         | index.html         | /index.html         |
-| api-guide.md     | api-guide.html     | /api-guide.html     |
-| about/license.md | about/license.html | /about/license.html |
+Source file      | Generated File     | URL Format
+---------------- | ------------------ | -------------------
+index.md         | index.html         | /index.html
+api-guide.md     | api-guide.html     | /api-guide.html
+about/license.md | about/license.html | /about/license.html
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:681]: Lets revert this change as it's not related to what this PR is initially targeting.

File: docs/user-guide/configuration.md [hunk 3]
```diff
@@ -668,34 +668,33 @@ setting `use_directory_urls` to `true` or `false`.
...
 
 For example, this needs to be set to `false` when:
 
-- opening pages directly from the file system
-- publishing the documentation to a static S3 website.
+* opening pages directly from the file system
+* publishing the documentation to a static S3 website.
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:686]: Same here, lets revert this change as it's not related to what this PR is initially targeting.

File: docs/user-guide/configuration.md [hunk 4]
```diff
@@ -668,34 +668,33 @@ setting `use_directory_urls` to `true` or `false`.
...
-| Source file      | Generated File            | URL Format     |
-| ---------------- | ------------------------- | -------------- |
-| index.md         | /index.html               | /              |
-| api-guide.md     | /api-guide/index.html     | /api-guide/    |
-| about/license.md | /about/license/index.html | /about/license |
+Source file      | Generated File            | URL Format
+---------------- | ------------------------- | --------------
+index.md         | /index.html               | /
+api-guide.md     | /api-guide/index.html     | /api-guide/
+about/license.md | /about/license/index.html | /about/license
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:697]: Again, lets revert this change as it's not related to what this PR is initially targeting.

**Review by pawamoy — CHANGES_REQUESTED** [review]
Thanks! Just a suggestion, and requests to revert stylistic-only changes.

**Comment by pawamoy** [comment]
Ah, wait, I posted review comments while looking at the last commit only. Let me update everything.

File: docs/user-guide/configuration.md [hunk 5]
```diff
@@ -661,25 +661,40 @@ option every time the `mkdocs serve` command is called.
...
-The following table demonstrates how the URLs used on the site differ when
+The following tables demonstrate how the directory structure and URLs used on the site differ when
 setting `use_directory_urls` to `true` or `false`.
 
-Source file      | use_directory_urls: true  | use_directory_urls: false
----------------- | ------------------------- | -------------------------
-index.md         | /                         | /index.html
-api-guide.md     | /api-guide/               | /api-guide.html
-about/license.md | /about/license/           | /about/license.html
+`use_directory_urls: false`
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:669]: Lets make this bold?

```suggestion
**`use_directory_urls: false`**
```

File: docs/user-guide/configuration.md [hunk 6]
```diff
@@ -661,25 +661,40 @@ option every time the `mkdocs serve` command is called.
...
+index.md         | index.html         | /index.html
+api-guide.md     | api-guide.html     | /api-guide.html
+about/license.md | about/license.html | /about/license.html
+
+For example, this needs to be set to `false` when:
+
+* opening pages directly from the file system
+* publishing the documentation to a static S3 website.
+
+`use_directory_urls: true`
```
Comment (pawamoy) [inline docs/user-guide/configuration.md:688]: Lets make this bold too?

```suggestion
**`use_directory_urls: true`**
```

**Review by pawamoy — CHANGES_REQUESTED** [review]
OK, updated my review :slightly_smiling_face:

**Comment by ronzulu** [comment]
Good point about the `/` instead of the `\`

**Review by pawamoy — APPROVED** [review]
Thanks for the update! Looking good :slightly_smiling_face:
